### PR TITLE
Updating Cray support for 2.7.0

### PR DESCRIPTION
**Pull request by gppezzi**
Preview is here: http://gppezzi-eb.readthedocs.org/en/cray/Cray_support.html

@boegel @pforai

### CODE REVIEW

**Comment by pforai** [comment]
Looks good as a first iteration, will give this a more detailed look in the evening and maybe a small PR.

File: docs/Cray_support.rst [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  * e.g., --optarch=sandybridge results in craype-sandybridge being loaded in the build environment used by EasyBuild 
+
+You can also export this option as a shell variable: Example for sandybridge::
+
+ $ export EASYBUILD_OPTARCH=sandybridge
+
+Metadata for Cray-provided modules
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Easybuild provides a sample metadata file in order to use modules provided by Cray:
```
Comment (gppezzi) [inline docs/Cray_support.rst]: @boegel can you show me how to add a link to your metadata doc?

Comment (boegel) [inline docs/Cray_support.rst]: @gppezzi use this (literally, including backticks)

```
:ref:`using_external_modules_metadata`
```

**Comment by boegel** [comment]
merged via #285, thanks @gppezzi

**Pull request merged**